packages/round-manager/src/features/program/CreateProgramPage.tsx

```diff
@@ -65,50 +73,117 @@ export default function CreateProgram() {
-            disabled={isLoading}
-            placeholder="Program name" />
-          <br />
-          <Input
-            {...register("operatorWallet1", { required: true })}
-            $hasError={errors.operatorWallet1}
-            type="text"
-            disabled={isLoading}
-            placeholder="Operator wallet 1"
-            defaultValue={account} />
-          <br />
-          <Input
-            {...register("operatorWallet2")}
-            type="text"
-            disabled={isLoading}
-            placeholder="Operator wallet 2" />
-          <br />
-          <Input
-            {...register("operatorWallet3")}
-            type="text"
-            disabled={isLoading}
-            placeholder="Operator wallet 3" />
-          <br />
-          <Button type="submit" disabled={isLoading || isSavingToIPFS || isSuccess}>
-            {isLoading || isSavingToIPFS ? "Creating..." : "Create"}
+        <div className="flow-root">
+          <h1 className="float-left text-[32px] mb-7">Create Grant Program</h1>
+          <Button
+            type="button"
+            $variant="outline"
+            className="inline-flex float-right py-2 px-4 text-sm text-grey-400"
+            onClick={() => navigate('/')}>
+              <XIcon className="h-5 w-5 mr-1" aria-hidden="true" />
+              Exit
           </Button>
+        </div>
+      </header>
+
+      <main className="grid md:grid-cols-3 gap-4">
+        <div>
+          <h2 className="text-base font-bold mb-4">Details</h2>
+          <h2 className="text-base text-grey-400">
+            Provide the name of the program as well as the round operators’ wallets addresses.
+          </h2>
+        </div>
+
+        <div className="col-span-2">
+
+          <form
+            className="grid grid-cols-1 gap-4 sm:items-start sm:border sm:border-gray-200 py-5 sm:px-10"
+            onSubmit={handleSubmit(onSubmit)}
+          >
+
+            <div className="sm:flex sm:flex-rows">
+              <div className="sm:basis-2/3">
+                <label htmlFor="name" className="block text-xs font-medium text-gray-700">Name</label>
+                <Input
+                  {...register("name", { required: true })}
+                  $hasError={errors.name}
+                  type="text"
+                  disabled={isLoading}
+                  placeholder="Program name"
+                />
+                {errors.name && <p className="text-sm text-red-600">{errors.name?.message}</p>}
+              </div>
+            </div>
+
+
+            <div>
+              <label htmlFor="operators" className="block text-xs font-medium text-gray-700">
+                Additional Operator Wallets
+              </label>
+
+              <ul>
+                {fields.map((item, index) => {
+                  return (
+                    <li key={item.id} className="flex flex-rows">
+                      <Input
+                        {...register(`operators.0.wallet`)}
```

Comment:
This is broken! Might need a hand @chibie to see why the index variable doesn't get resolved
I've commented out the code below to get it working for now

Reply:
i'll take a look